app/services/article_api_index_service.rb

```diff
@@ -74,18 +74,29 @@ def tag_articles
   end
 
   def top_articles
-    Article.published.order("positive_reactions_count DESC").where("published_at > ?", top.to_i.days.ago).
+    Article.published.includes(:user, :organization).
+      where("published_at > ?", top.to_i.days.ago).
+      order("positive_reactions_count DESC").
       page(page).per(per_page || DEFAULT_PER_PAGE)
   end
 
   def state_articles(state)
-    if state == "fresh"
-      Article.published.
-        where("positive_reactions_count < ? AND featured_number > ? AND score > ?", 2, 7.hours.ago.to_i, -2)
-    elsif state == "rising"
-      Article.published.
-        where("positive_reactions_count > ? AND positive_reactions_count < ? AND featured_number > ?", 19, 33, 3.days.ago.to_i)
-    end
+    articles = Article.published.includes(:user, :organization)
```

Comment:
we weren't loading those two relations that are accessed in the JSON rendering for the API